encoding/codecv6.go

```diff
@@ -0,0 +1,77 @@
+package encoding
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/scroll-tech/go-ethereum/crypto/kzg4844"
+)
+
+type DACodecV6 struct {
+	DACodecV4
+}
+
+// Version returns the codec version.
+func (d *DACodecV6) Version() CodecVersion {
+	return CodecV6
+}
+
+// MaxNumChunksPerBatch returns the maximum number of chunks per batch.
+func (d *DACodecV6) MaxNumChunksPerBatch() int {
+	return 1
```

Comment:
Why 1? Just a placeholder?

Reply:
Since `Chunks` are not exposed to DA, there should be no details about chunks here

Reply:
In that case, `rollup-relayer` should not use `MaxNumChunksPerBatch`, right? Otherwise we'll keep producing batches with a single chunk.

Reply:
yeah might be better to return `maxInt` here instead. then we can configure a lower target in the relayer

Reply:
done in 1dde89ae3ab03889645a294ecb35afc549ceef1f